resources/views/account/partials/queries/orders.graphql

```diff
@@ -0,0 +1,23 @@
+{
+  customer {
+    orders {
+      items {
+        number
+        order_date
+        status
+        shipping_address {
+          firstname
+          lastname
+        }
+        total {
+          grand_total {
+            value
+          }
+        }
+        items {
+          product_name
+        }
+      }
+    }
+  }
+}
```

Comment:
This one is very similair to the default rapidez orders.graphql but that one didn't include the amount of products on the orders overviewpage, this was needed to show a counter for amount of products in an order

Reply:
PR it there so we've this everywhere